sdk/python/feast/feature_service.py

```diff
@@ -34,13 +41,10 @@ def __init__(
         tags: Optional[Dict[str, str]] = None,
     ):
         """
-        Create a new Feature Service object.
+        Inits the FeatureService class.
```

Comment:
`Inits` is not really descriptive. `Creates`?

Reply:
That's fair, I'll leave this docstring as is (for context, I took this inits docstring from the Google style guide, but I agree it's pretty non-descriptive).